src/components/NavMenu.tsx

```diff
@@ -45,13 +45,17 @@ export const NavMenu = () => {
     <div className="absolute bottom-4 left-1/2 z-50 flex -translate-x-1/2 gap-x-2">
       <ul className="menu rounded-box menu-horizontal bg-base-200">
         {LINKS.map((link) => (
-          <li
-            key={link.href}
-            className={cn("", {
-              "text-accent": activePage === link.href.slice(1),
-            })}
-          >
-            <Link href={link.href}>{link.icon}</Link>
+          <li key={link.href} className={cn("bg-transparent")}>
+            <div>
```

Comment:
The fix is to wrap it in any sort of container because the CSS from `menu` parent class applies a background color and text color to the direct child of the `<li></li>` if it is an element with an onClick event (e.g. buttons and anchors).

If instead of a button or anchor link, it were a `<p></p>` element, the same behavior wouldn't be observed.

Reply:
Changed it to a span.